packages/ui-select/src/Select/index.tsx

```diff
@@ -656,25 +656,28 @@ class Select extends Component<Props> {
   // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'data' implicitly has an 'any' type.
   renderList(data) {
     const { getListProps, getOptionProps, getDisabledOptionProps } = data
-    const { isShowingOptions, optionsMaxWidth, visibleOptionsCount, children } =
-      this.props
+    const {
+      isShowingOptions,
+      optionsMaxWidth,
+      visibleOptionsCount,
+      children
+    } = this.props
 
     let lastWasGroup = false
     const viewProps = isShowingOptions
       ? {
-          display: 'block',
-          overflowY: 'auto',
+          display: 'block' as const,
```

Comment:
I'm not sure why this is needed here... I mean I can see it is needed, just don't completely understand how this works, can you please explain later? :)

Reply:
I added this because of some strange TS error, but it has gone away, so this is not needed anymore, deleted.

Reply:
Weird :D Thanks!